repo_blob.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package git
+
+// CatFileBlob returns the blob corresponding to the given revision of the repository.
+func (repo *Repository) CatFileBlob(rev string) (*Blob, error) {
+	revHash, err := r.RevParse(rev, RevParseOptions{Timeout: opt.Timeout}) //nolint
```

Comment:
```suggestion
	rev, err := r.RevParse(rev, RevParseOptions{Timeout: opt.Timeout}) //nolint
```

You can reuse the same variable.